tests/realtikvtest/testkit.go

```diff
@@ -141,6 +142,7 @@ func CreateMockStoreAndDomainAndSetup(t *testing.T, opts ...mockstore.MockTiKVSt
 	t.Cleanup(func() {
 		dom.Close()
 		require.NoError(t, store.Close())
+		infosync.MockGlobalServerInfoManagerEntry.Close()
```

Comment:
I see this is used in some places. Could we put it to `dom.Close()` or some other way to keep it from being called in more than one place?